cal/cal/models.py

```diff
@@ -584,6 +584,21 @@ def query(self, calendar_ids=None, start=None, end=None):
         return events_qs
 
 
+    def queryWeek(self, calendar_ids=None, start=None, end=None):
```

Comment:
It would be helpful to write a method comment describing what this function does, since it is not easy to tell from all the iteration code below